src/main/resources/Ideas/IdeasTranslations.xml

```diff
@@ -67,7 +67,7 @@ ideas.livetable.priority=Priority
 
 #Ideas application keys
 ideas.home.description=Ideas worth spreading
-ideas.message.guess=Guess can't vote
+ideas.message.guess=Guest can't vote
```

Comment:
Don`t you mean "Guests can't vote" (plural)?

If you would use a singular, the you would have to articulate, i.e. "The/A guest can't vote".